src/teacup.coffee

```diff
@@ -174,6 +176,13 @@ class Teacup
     @raw contents
     @raw "</#{tagName}>"
 
+  scriptTag: (tagName, args...) ->
+    {attrs, contents} = @normalizeArgs args
+    @raw "<#{tagName}#{@renderAttrs attrs}>"
+    @raw @scriptescape contents
```

Comment:
I suppose this is ok if we remember to stringify any dirty content into the script body.